src/documentation/markdown/2017/lar-data-fields.md

```diff
@@ -0,0 +1,378 @@
+  - 2 - Asian or Pacific Islander
+  - 3 - Black or African American
+  - 4 - Hispanic
+  - 5 - White
+  - 6 - Information not provided by applicant in mail, internet, or telephone application
+  - 7 - Not applicable
+
+### [applicant\_race\_4](#applicant_race_4)
+
+- **Description:** Race of the applicant or borrower
+- **Values:**
+  - 1 - American Indian or Alaska Native
+  - 2 - Asian or Pacific Islander
+  - 3 - Black or African American
+  - 4 - Hispanic
+  - 5 - White
+  - 6 - Information not provided by applicant in mail, internet, or telephone application
+  - 7 - Not applicable
+
+### [applicant\_race\_5](#applicant_race_5)
+
+- **Description:** Race of the applicant or borrower
+- **Values:**
+  - 1 - American Indian or Alaska Native
+  - 2 - Asian or Pacific Islander
+  - 3 - Black or African American
+  - 4 - Hispanic
+  - 5 - White
+  - 6 - Information not provided by applicant in mail, internet, or telephone application
+  - 7 - Not applicable
+
+### [co\_applicant\_race\_1](#co_applicant_race_1)
+
+- **Description:** Race of the first co-applicant or co-borrower
+- **Values:**
+  - 1 - American Indian or Alaska Native
+  - 2 - Asian or Pacific Islander
+  - 3 - Black or African American
+  - 4 - Hispanic
+  - 5 - White
+  - 6 - Information not provided by applicant in mail, internet, or telephone application
+  - 7 - Not applicable
+  - 8 - No co-applicant
+
+### [co\_applicant\_race\_2](#co_applicant_race_2)
+
+- **Description:** Race of the first co-applicant or co-borrower
+- **Values:**
+  - 1 - American Indian or Alaska Native
+  - 2 - Asian or Pacific Islander
+  - 3 - Black or African American
+  - 4 - Hispanic
+  - 5 - White
+
+### [co\_applicant\_race\_3](#co_applicant_race_3)
+
+- **Description:** Race of the first co-applicant or co-borrower
+- **Values:**
+  - 1 - American Indian or Alaska Native
+  - 2 - Asian or Pacific Islander
+  - 3 - Black or African American
+  - 4 - Hispanic
+  - 5 - White
+
+### [co\_applicant\_race\_4](#co_applicant_race_4)
+
+- **Description:** Race of the first co-applicant or co-borrower
+- **Values:**
+  - 1 - American Indian or Alaska Native
+  - 2 - Asian or Pacific Islander
+  - 3 - Black or African American
+  - 4 - Hispanic
+  - 5 - White
+
+### [co\_applicant\_race\_5](#co_applicant_race_5)
+
+- **Description:** Race of the first co-applicant or co-borrower
+- **Values:**
+  - 1 - American Indian or Alaska Native
+  - 2 - Asian or Pacific Islander
+  - 3 - Black or African American
+  - 4 - Hispanic
+  - 5 - White
+
+### [applicant\_sex](#applicant_sex)
+
+- **Description:** Sex of the applicant or borrower
+- **Values:**
+  - 1 - Male
+  - 2 - Female
+  - 3 - Information not provided by applicant in mail, internet, or telephone application
+  - 4 - Not applicable
+  - 6 - Applicant selected both male and female
+
+### [co\_applicant\_sex](#co_applicant_sex)
+
+- **Description:** Sex of the first co-applicant or co-borrower
+- **Values:**
+  - 1 - Male
```

Comment:
for 2017:
1.Male
2.Female
3.Information not provided by applicant in
mail, Internet, or telephone application
(see App. A, I.D.2.)
4.Not applicable
5.No co-applicant